libs/designsystem/src/lib/index.ts

```diff
@@ -5,4 +5,6 @@ export * from './helpers';
 export * from './scss/scss-helper';
 // export * from './testing';
 
+export { KirbyBadge as BadgeComponent } from './components/web-component-proxies.component';
```

Comment:
Perhaps add a comment stating that everything will go wrong and Cthulhu will be summoned if badge is not exported this way.

Reply:
Fixed in 99f0a74